### PR TITLE
HotFix: fixed getting secrets issue

### PR DESCRIPTION
**Pull request by wannacfuture**
Resolves https://github.com/ubiquibot/QA-Improvements/issues/1

### CODE REVIEW

File: .github/workflows/qa-automation.yml [hunk 1]
```diff
@@ -1,39 +1,35 @@
 name: Automate QA
 
 on:
-  pull_request:
+  pull_request_target:
```
Comment (rndquu) [inline .github/workflows/qa-automation.yml:4]: `pull_request_target` + `actions/checkout@v3` allows secrets exfiltrating

https://medium.com/tinder/exploiting-github-actions-on-open-source-projects-5d93936d189f

I think we can simply remove the `actions/checkout@v3` step

Comment (wannacfuture) [reply .github/workflows/qa-automation.yml:4]: okay, fixed

File: .github/workflows/qa-automation.yml [hunk 2]
```diff
@@ -1,39 +1,32 @@
...
-        uses: tibdex/github-app-token@v1.7.0
-        id: get_installation_token
-        with:
-          app_id: ${{ secrets.UBIQUITY_BOUNTY_BOT_APP_ID }}
-          private_key: ${{ secrets.UBIQUITY_BOUNTY_BOT_PRIVATE_KEY }}
+    permissions:
+      contents: read
 
-      - name: add pr to project
-        if: contains(github.event.pull_request.requested_reviewers.*.login, 'Draeieg')
```
Comment (0xcodercrane) [inline .github/workflows/qa-automation.yml:25]: This is actually needed! would you re-add this check?

**Review by 0xcodercrane — CHANGES_REQUESTED** [review]

File: .github/workflows/qa-automation.yml (hunk 2 above)
Comment (wannacfuture) [reply .github/workflows/qa-automation.yml:25]: fixed

**Pull request closed without merging**